src/modules/helpchan.ts

```diff
@@ -115,13 +115,7 @@ export class HelpChanModule extends Module {
 	occupiedEmbed(asker: GuildMember) {
 		return new MessageEmbed(this.OCCUPIED_EMBED_BASE)
 			.setDescription(occupiedMessage(asker))
-			.setFooter(
-				`Closes after ${
-					dormantChannelTimeout / 60 / 60 / 1000
-				} hours of inactivity or when ${
-					asker.displayName
-				} sends !close.`,
-			);
+			.setFooter(`Closes after ${dormantChannelTimeoutHours} hours of inactivity or when ${asker.username} sends !close.`);
```

Comment:
I changed this to displayName on purpose, as it's easier to recognize someone by that

Reply:
Ahh sorry, I merged master into this branch and didn't notice that change.

Reply:
Fixed in https://github.com/typescript-community/community-bot/pull/132/commits/e7b1135c4b49eba6e2a81f9d57a18f35499161ec